x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/details_page/details_page_mappings_content/mappings_information_panels.tsx

```diff
@@ -47,20 +48,33 @@ export const MappingsInformationPanels = ({
   refetchMapping,
 }: MappingsInformationPanelsProps) => {
   const {
-    plugins: { cloud },
+    plugins: { cloud, licensing },
     core: { application },
   } = useAppContext();
   const state = useMappingsState();
 
+  const [isEnterpriseLicense, setIsEnterpriseLicense] = useState<boolean>(false);
+  const [isUpdatingElserMappings, setIsUpdatingElserMappings] = useState<boolean>(false);
+
   const showAboutMappingsStyles = css`
     ${useEuiBreakpoint(['xl'])} {
       max-width: 480px;
     }
   `;
+
   const hasSemanticText = hasSemanticTextField(state.mappingViewFields);
   const hasElserOnMlNodeSemanticText = hasElserOnMlNodeSemanticTextField(state.mappingViewFields);
+  const shouldShowEisUpdateCallout =
+    (cloud?.isCloudEnabled && (isEnterpriseLicense || cloud?.isServerlessEnabled)) ?? false;
+
+  useEffect(() => {
+    const subscription = licensing?.license$.subscribe((license: ILicense) => {
```

Comment:
nit: you don't need to create a subscription, you can also call `async getLicense()` from the start object, which will grab the license from the Observable. Or you might be able to use `useObservable(licensing?.license$)`. But with`useObservable` you might have to create an empty observable if `licensing` is `undefined`.